azure-pipelines.yml

```diff
@@ -44,6 +45,18 @@ schedules:
     - main
 
 jobs:
+  - job: build_windows_samples
+    displayName: Build Windows Samples
+    pool:
+      vmImage: windows-2019
+    steps:
+      - task: MSBuild@1
+        displayName: Build Solution
```

Comment:
only build solution, we do not need to run tests, etc

Reply:
@jfversluis is this change ok to you?

Reply:
Yep perfect, I have been thinking about that myself :)